assets/src/components/notificationModals/chelseaLoweredNotificationModal.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from "react"
+import { StateDispatchContext } from "../../contexts/stateDispatchContext"
+import { closeIcon } from "../../helpers/icon"
+import { setNotification } from "../../state"
+
+const ChelseaLoweredNotificationModal = () => {
+  const [, dispatch] = useContext(StateDispatchContext)
+
+  const closeModal = () => {
+    dispatch(setNotification(undefined))
+  }
```

Comment:
**issue:** most of this component doesn't need tests but the close interaction should get one.

Reply:
fixed in 1213e1a